### PR TITLE
Fix tests and test with Django 1.10

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: htmlvalidator/tests/urls.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-urlpatterns = patterns(
-    '',
+urlpatterns = [
     url(r'^view/', views.html_view, name='html_view')
-)
+]
+try:
+    from django.conf.urls import patterns
+    urlpatterns = patterns('', *urlpatterns)
+except:  # Django >= 1.10
```
Comment (peterbe) [inline htmlvalidator/tests/urls.py]: Can you instead say what exception we expect. E.g. `except ImportError:`

**Comment by peterbe** [comment]
Great work!

File: htmlvalidator/tests/urls.py (hunk 1 above)
Comment (scop) [inline htmlvalidator/tests/urls.py]: Sure, done/amended.

**Comment by peterbe** [comment]
Version 0.4.6 should be available on pypi soon.

**Pull request merged**